### ISSUE TITLE
Support disabled attribute

### ISSUE THREAD

**Issue by jzaefferer** [AmpersandJS/ampersand-checkbox-view#13]
It should be possible to make a checkbox disabled. Is there a reason for this one to not include that? Or just missing?

**Comment by jzaefferer** [comment]
Nevermind, `type="checkbox"` doesn't support readonly attribute.

**Comment by jzaefferer** [comment]
Reopened and renamed. Disabled attribute is supported, but not implemented by this view.

**Comment by pgilad** [comment]
@doratias What do you think?

**Comment by wraithgar** [comment]
What api are we thinking then? `.enable()` and `.disable()`?  Seems like a very sensible thing to support, in any case.

**Comment by cdaringe** [comment]
hi @jzaefferer, all view's accept this, and do so through the `bindings` object.  i made a fiddle to demonstrate: http://requirebin.com/?gist=f6027f76d0d84d47a1ca

there have been a lot of discussions about which view modules should accept which DOM attrs/props.  As a general rule, we've been somewhat defaulting to saying 'use the bindings' vs. add additional code for single attributes.

HTH

**Comment by jzaefferer** [comment]
Well, having to extend the view isn't exactly "accepting" (or even supporting). Considering all the existing supported properties, excluding disabled seems pretty arbitrary.

If that's what its going to be, I sugges to at least improve the documentation: https://github.com/AmpersandJS/ampersand-checkbox-view#properties (pointing to source code in place of proper docs isn't that great either).

**Comment by cdaringe** [comment]
@jzaefferer, i agree it's verbose.  it is certainly 'supported', however. :)  definitely accepting PRs to update as you think would increase understanding and clarity.

**Comment by jzaefferer** [comment]
Sorry, I won't be able to help with a PR.
